src/_P020_Ser2Net.ino

```diff
@@ -288,6 +288,34 @@ boolean Plugin_020(uint8_t function, struct EventStruct *event, String& string)
         success = true;
       }
 
+      if (command == F("serialsendhex")){
+        success = true;
+        String tmpString = string.substring(14);
+        unsigned int len = tmpString.length();
+        const char* buf = tmpString.c_str();
+        for(unsigned int i=0; i < len / 2; i++){
+          uint8_t uc = buf[i * 2], dc = buf[i * 2 + 1];
```

Comment:
For readability it would be nice to allow optional spaces and `0x` prefixes, that's currently not possible/allowed

Reply:
If spaces are allowed, then the argument must be wrapped in quotes. (and `parseString` must be used)

Reply:
Or use `parseStringToEnd` so no quotes are required, but still allowed 😃